internal/pkg/validator/validator.go

```diff
@@ -42,29 +48,14 @@ type Rule struct {
 
 type contextKey string
 
-// Validate nested struct fields or slice items.
-func Validate(ctx context.Context, value interface{}, rules ...Rule) error {
-	return ValidateCtx(ctx, value, "dive", "", rules...)
-}
-
-// ValidateCtx validates any value.
-func ValidateCtx(ctx context.Context, value interface{}, tag string, namespace string, rules ...Rule) error {
-	v := newValidator()
```

Comment:
Pri kazdej validacii vytvarame novy validator, pri ladeni Templates API a lokalnom profilovaní som zistil,
ze to zabera `5-6%` CPU time pri vacsich requestoch.

Preto sa to oplati optimalizovat.
Nie vsade, ale pri nacitavani local/remote state, kde sa validuje kazdy objekt.

Reply:
Tj. zmazal som tieto public funkcie a pridal public funkciu `New`, ktora vrati validator s tymito funkciami.
Tak to nemusi registrovat pravidla a preklady (nase hlasky) pri kazdej validacii znova.